demo/src/main/res/drawable/thumb.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item>
```

Comment:
Not sure why you need a fully transparent oval?

Reply:
I think I added this for the padding but you're right, this is redundant 😄

Reply:
1f646cb4ad6185e55e1ac1327cfd37b86c4cafef